frontend/src/components/maps/leaflet/InfoSlideOut/BuildingAttributes.tsx

```diff
@@ -10,13 +10,23 @@ import { ThreeColumnItem } from './ThreeColumnItem';
 interface IBuildingAttributes {
   /** the selected building information */
   buildingInfo: IBuilding;
+  /** whether the user has the correct agency/permissions to view all the details */
+  canViewDetails: boolean;
+  /** whether the user has the correct agency/permissions to edit property details */
+  canEditDetails: boolean;
 }
 
 /**
  * Displays Building specific information needed on the information slide out
  * @param buildingInfo the selected parcel data
+ * @param canViewDetails user can view all property details
+ * @param canEditDetails user can edit property details
  */
-export const BuildingAttributes: React.FC<IBuildingAttributes> = ({ buildingInfo }) => {
+export const BuildingAttributes: React.FC<IBuildingAttributes> = ({
```

Comment:
So you've added the can*Details to BuildingAttributes, but aren't using it, which doesn't seem right given that the code was previously hiding the BuildingAttributes if the user didn't have view permissions.

Reply:
i'm just going to add my changes for the building slide out to here as well, this is getting too confusing